.moban.dt/local-README.rst.jj2

```diff
@@ -8,8 +8,10 @@
 It is used with `yehua <https://github.com/chfw/yehua>`_.
 Organisations using it:
 
-- `pyexcel <https://github.com/pyexcel/pyexcel>`_.
-- `coala <https://github.com/coala/coala>`_.
+- `pyexcel mobans <https://github.com/pyexcel/pyexcel-mobans>`_.
+- `coala mobans<https://gitlab.com/coala/mobans>`_.
```

Comment:
A space is missing between `mobans` and `<`. Links will not properly render. ([Preview](https://github.com/moremoban/pypi-mobans/blob/825fc08af2c993673030163e901dccb1b26cfd01/README.rst))
```suggestion
- `coala mobans <https://gitlab.com/coala/mobans>`_.
```

Reply:
true